_includes/footer.html

```diff
@@ -5,9 +5,9 @@
         <ul class="text-left">
           <li>&copy; {{ site.time | date: '%Y' }} <span>GitHub</span>, Inc.</li>
           <li><a href="https://github.com/site/terms">Terms</a></li>
-          <li><a href="https://training.github.com/kit/license.html">License</a></li>
+          <li><a href="{{ site.baseurl }}/license.html">License</a></li>
```

Comment:
Thanks for checking! `{{ site.baseurl }}` points to the kit in particular, not services.